cli.js

```diff
@@ -1582,8 +1582,11 @@ async function execute(network, action, ...params) {
     const minHealthFactor = web3.utils.toWei(params[1]);
     const targetHealthFactor = web3.utils.toWei(params[2]);
     const maxHealthFactor = web3.utils.toWei(params[3]);
+    const borrowToken = param[4];
+    const collateralToken = param[5];
+    const rateMode = param[6];
```

Comment:
this should be redone as in other places in the CLI. Commands should accepts token symbols instead of addresses, and the rate mode should be variable or stable instead of 1 or 2.